package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dotnet-build",
-    "version": "4.2.0",
+    "version": "5.0.0",
```

Comment:
You must've liked this... 😄

Reply:
I liked it, that is mostly why I did it. But also because I think it could be a breaking change.